en/getting_started/README.md

```diff
@@ -37,10 +37,14 @@ To get the source files:
 #### Install Qt
 
 You **need to install Qt as described below** instead of using pre-built packages from say, a Linux distribution, because *QGroundControl* needs access to private Qt headers.
-1. Download the [Qt installer](http://www.qt.io/download-open-source) (version {{ book.qt_version }}). You will also need to install the *Qt Speech* package.
-   - **Ubuntu:** Set the downloaded file to executable using:`chmod +x`. Install to default location for use with **./qgroundcontrol-start.sh.** If you install Qt to a non-default location you will need to modify **qgroundcontrol-start.sh** in order to run downloaded builds.
-   - **Windows:** Make sure to install *VS 2015 32 bit* package.
 
+To install Qt:
+1. Download and run the [Qt Online Installer](http://www.qt.io/download-open-source)
```

Comment:
@DonLakeFlyer This originally also said
```
   - You will also need to install the *Qt Speech* package.
```
But this package is not separately referenced in the installer. Does this need to be said (ie is this installed separately or is it now in the "core")?

Reply:
No longer needed in the new QT version we are using